modules/4337/package.json

```diff
@@ -47,15 +47,15 @@
     "url": "https://github.com/safe-global/safe-modules/issues"
   },
   "devDependencies": {
-    "@account-abstraction/contracts": "^0.7.0",
+    "@account-abstraction/contracts": "0.7.0",
```

Comment:
Should this be a dependency? I think we import types from it when building the contract:

https://github.com/safe-global/safe-modules/blob/bb5e1f7360633d5ecfb175c5d8667b2a3a1b746d/modules/4337/contracts/Safe4337Module.sol#L6-L9

Reply:
Yes sir.